components/constellation/constellation.rs

```diff
@@ -3609,14 +3597,9 @@ where
         size_type: WindowSizeType,
         browsing_context_id: BrowsingContextId,
     ) {
-        // TODO(mandreyel): can we avoid the second lookup?
         if let Some(browsing_context) = self.browsing_contexts.get_mut(&browsing_context_id) {
```

Comment:
What do you think about eliminating the second browsing context lookup? I forgot to include this in my original commit.